src/modules/ics/ics.controller.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+  Res,
+} from '@nestjs/common';
+import { IcsService } from './ics.service';
+import type { Response } from 'express';
+
+@Controller('ics')
+export class IcsController {
+  constructor(private readonly icsService: IcsService) {}
+
+  @Get(':eventId.ics')
+  async getIcsFile(@Param('eventId') eventId: number, @Res() res: Response) {
```

Comment:
Trả về toàn bộ events


```suggestion
  @Get('calendar.ics')
  async getIcsFile(@Res() res: Response) {
```